src/plugins/intel_gpu/tests/test_utils/test_utils.cpp

```diff
@@ -306,8 +306,12 @@ cldnn::engine& get_test_engine() {
 cldnn::stream_ptr get_test_stream_ptr() {
     static std::shared_ptr<cldnn::stream> test_stream = nullptr;
     if (!test_stream) {
+#ifdef ENABLE_ONEDNN_FOR_GPU
+        ExecutionConfig cfg(ov::intel_gpu::queue_type(QueueTypes::in_order));
```

Comment:
@byungilm  is this OK? I think this is not a correct fix..

Reply:
Removed. I think setting for in_order should move into each test-case.